guides/Getting Started/faq.md

```diff
@@ -51,7 +51,11 @@ const { token } = require('./config.json');
 
 For a tutorial
 
-1. Set `process.env.DISCORD_TOKEN` equal to your bot's token, you can do this using [dotenv](https://www.npmjs.com/package/dotenv). Run `npm install dotenv --save`
+Set `process.env.DISCORD_TOKEN` equal to your bot's token, you can do this by either
```

Comment:
This removed the number. Needs to be fixed.

Reply:
I removed the number because I felt like that didn't need to be an ordered list. Feel free to make any changes you want!